service.libraryautoupdate/addon.xml

```diff
@@ -0,0 +1,87 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>

```

Comment:
please change the `CRLF` line endings to `LF` line endings in this file

Reply:
Ok, should be done.